data/update.py

```diff
@@ -141,41 +117,40 @@ def run(options):
 
 # Upload the scan + processed data to /live/ and /archive/ locations by date.
 def upload(date):
-  live_scanned = "s3://%s/live/scan/" % (BUCKET_NAME)
-  live_cached = "s3://%s/live/cache/" % (BUCKET_NAME)
-  live_db = "s3://%s/live/db/" % (BUCKET_NAME)
+  live_parents = "s3://%s/live/parents/" % (BUCKET_NAME)
   live_subdomains = "s3://%s/live/subdomains/" % (BUCKET_NAME)
-  archive_scanned = "s3://%s/archive/%s/scan/" % (BUCKET_NAME, date)
-  archive_cached = "s3://%s/archive/%s/cache/" % (BUCKET_NAME, date)
-  archive_db = "s3://%s/archive/%s/db/" % (BUCKET_NAME, date)
+  live_db = "s3://%s/live/db/" % (BUCKET_NAME)
   archive_subdomains = "s3://%s/archive/%s/subdomains/" % (BUCKET_NAME, date)
+  archive_parents = "s3://%s/archive/%s/parents/" % (BUCKET_NAME, date)
+  archive_db = "s3://%s/archive/%s/db/" % (BUCKET_NAME, date)
```

Comment:
This S3 uploading code is not fully tested yet, but given the weight of this PR, I think it's okay to merge this as-is and then we can file a new PR fixing any issues here.

Reply:
Concur